### PR TITLE
remove PATENTS, move font to third_party

### PR DESCRIPTION
**Pull request by maryx**
* Removes PATENTS
 * Removes the OFL license text from the LICENSE
 * Adds a NOTICES file per project (not used; we don't distribute binaries, and nor does Flutter code currently recognize the NOTICES file). The NOTICES file contains 1) LICENSE, 2) OFL
 * Moves Raleway font and license to root /third_party dir
 * Copies Raleway font and license to /third_party inside each subproject that uses it

### CODE REVIEW

File: unit_converter/unit_converter/pubspec.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ flutter:
   fonts:
    - family: Raleway
      fonts:
-      - asset: assets/fonts/Raleway-Regular.ttf
+      - asset: ../../third-party/Raleway-Regular.ttf
```
Comment (maryx) [inline unit_converter/unit_converter/pubspec.yaml]: @Hixie I'd like some clarification about how to refer to a font saved in /third_party/ when my pubspec.yaml is located in /unit_converter/unit_converter/

I tried to use a relative path but received the error:
`Error: unable to locate asset entry in pubspec.yaml: "../../third-party/Raleway-Regular.ttf".`

Thanks!

**Comment by Hixie** [comment]
The conclusion I think was that we'd copy the font into each project right?

**Comment by maryx** [comment]
Correct, and add a NOTICES file. I will update this PR today or by Monday.

**Comment by Hixie** [comment]
Don't do the NOTICES thing yet for the file with the two licenses, we'll have to fix that on our end first. Just use LICENSE for now as before. The framework doesn't yet know about NOTICES.

**Comment by Hixie** [comment]
LGTM modulo my last comment.

**Comment by maryx** [comment]
@Hixie I'm a little confused by `modulo my last comment`, so to confirm:
- This repo's projects don't need a NOTICES file because we're not distributing binaries
- However, I'm including a NOTICES file anyway
- The framework won't be aware of this NOTICES file. This is fine because we don't show a License screen in the app at the moment

Let me know if that SGTY.

**Pull request merged**